src/common/Element.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Base class for all elements
+ */
+import Props from "../common/Props.js";
+import defineFormAssociated from "./defineFormAssociated.js";
+import defineEvents from "./defineEvents.js";
+
+const Self = class NudeElement extends HTMLElement {
+	static postConstruct = [];
+	#initialized = false;
+
+	constructor () {
+		super();
+		this.constructor.init();
+
+		if (this.propChangedCallback && this.constructor.props) {
+			this.addEventListener("propchange", this.propChangedCallback);
+		}
+
+	}
+
+	connectedCallback () {
+		if (!this.#initialized) {
+			this.initializeProps?.();
+
+			this.#initialized = true;
+		}
+	}
```

Comment:
For some reason, this code is not reachable. Because of this, `<color-slider>` is broken.

Reply:
Should we call `super.connectedCallback()` from the subclasses?

Reply:
Is it legal at all? :)

Reply:
> Should we call `super.connectedCallback()` from the subclasses?

Oh yes, absolutely. I must have forgotten